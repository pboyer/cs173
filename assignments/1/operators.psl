print(<(3,4));
print("\n");
print(>(4,5));
print("\n");
print(+(3,4,5));
print("\n");
print(-(9,8,7));
print("\n");
print(+("one ", "two ", "three"));
print("\n");
print(==(5,5));
print("\n");
print(==("foo",5));
print("\n");
print(==({x:5},5));
print("\n");
print(==({x:5},{x:5}));
print("\n");
print(==({x:5},{x:6}));
print("\n");
print(==({x:5},{y:6}));
print("\n");
==({x:5},{y:5});