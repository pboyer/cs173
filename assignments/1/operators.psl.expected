true
false
12
-6
one two three
true
false
false
true
false
false
false
